trust-quorum/src/coordinator_state.rs

```diff
@@ -49,6 +52,7 @@ impl CoordinatorState {
     /// Return the newly constructed `CoordinatorState` along with this node's
     /// `PrepareMsg` so that it can be persisted.
     pub fn new_uninitialized(
+        log: Logger,
```

Comment:
Should this add a component context via `log.new` indicating that it is the `coordinator_state` component?

Reply:
Done in [50ebf64](https://github.com/oxidecomputer/omicron/pull/8000/commits/50ebf6479db5b3e8e345394a1bcf2943306edf78)